src/core/common/utils.ts

```diff
@@ -667,6 +667,8 @@ export function setFormatOptions(options: FormatLocaleDefinition) {
   };
 }
 
+export const tickFormatParserExpression = /\{([^}]+)\}/g;
```

Comment:
I believe that using a single regex instance may retain its own state, can you verify that this will work across multiple calls?

Reply:
Perhaps it might be better wrapped in a function so that a new regex object is created for each call.